lib/scientist/experiment.rb

```diff
@@ -190,29 +190,18 @@ def run(name = nil)
     behaviors.freeze
     context.freeze
 
-    name = (name || "control").to_s
+    name = (name || :control).to_s
```

Comment:
Is this related to your extraction? If not, please leave it as a String in this diff.